mera/utils.py

```diff
@@ -0,0 +1,133 @@
+import pandas as pd
+from typing import Optional
```

Comment:
Imports order, typing is a built-in module, so should be above pandas

Reply:
this should have been auto-sorted by isort, maybe you don't have it configured properly?